sm64.ld

```diff
@@ -315,10 +306,18 @@ SECTIONS
 
    /* load YAY0 and level data */
    YAY0_SEG(segment2, 0x02000000)
-#ifdef VERSION_EU
-   YAY0_EU_SEG(translation_en, 0x19000000)
-   YAY0_EU_SEG(translation_fr, 0x19000000)
-   YAY0_EU_SEG(translation_de, 0x19000000)
+
+#ifdef MULTILANG
+   YAY0_SEG(translation_en, 0x19000000)
+#ifdef ENABLE_FRENCH
+   YAY0_SEG(translation_fr, 0x19000000)
+#endif
+#ifdef ENABLE_GERMAN
+   YAY0_SEG(translation_de, 0x19000000)
+#endif
+#ifdef ENABLE_JAPANESE
+   YAY0_SEG(translation_jp, 0x19000000)
+#endif
```

Comment:
just a note for later, we should probably find a place to define these addresses like `0x19000000`